code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -667,12 +662,18 @@ obj/structure/machinery/cm_vending/sorted/uniform_supply
 		list("Heat Absorbent Coif", 10, /obj/item/clothing/mask/rebreather/scarf, VENDOR_ITEM_REGULAR),
 		)
 
+/obj/structure/machinery/cm_vending/sorted/uniform_supply/ui_state(mob/user)
+	return GLOB.not_incapacitated_and_adjacent_strict_state
+
 /obj/structure/machinery/cm_vending/sorted/uniform_supply/populate_product_list(var/scale)
 	return
 
 /obj/structure/machinery/cm_vending/sorted/uniform_supply/vend_fail()
 	return
 
+/obj/structure/machinery/cm_vending/sorted/uniform_supply/ui_state(mob/user)
+	return GLOB.not_incapacitated_and_adjacent_state
+
```

Comment:
duped proc that should be inherited? I'm seeing a few /ui_state procs that look extraneous

Reply:
It's good in this case. Removing the suplicate definition here.